addons/sale_timesheet/controllers/main.py

```diff
@@ -18,13 +18,41 @@ def plan(self, domain):
         }
 
     def _prepare_plan_values(self, domain):
+
+        def float_to_time(hour_number):
+            minutes = hour_number * 60
+            return _('%sh%s') % (str(int(minutes/60)), '%02d' % int(minutes % 60))
```

Comment:
1/ What about lang-based formatting ?
2/ you may use divmod: hours, minuts = divmod(hour_number * 60, 60)
3/ resource already have a float to time that effectively returns a time; here it is more a float_to_hours_str or simething like that